### DISCUSSION TITLE
Resource-List/Table or Widget inside infolist?

### DISCUSSION THREAD

**Discussion by RPotkowaVegasystems** [filamentphp/filament#7836] [Help]
What is the best approach to achive somithing like that:

Infolist of the Resouce "User" with 3 Tabs.
In one Tab should go a ResourceList/Table or Widget with all the related "Resource->Contact" Contacts from this User.

Its like the RelationManager but in a view and custom placement inside the infolist.

**Comment by pboivin** [comment]
I'm not completely sure but I think you'll be able to embed a Table inside of an Infolist Tab via a custom Livewire component.
